onnxruntime/test/optimizer/nhwc_transformer_test.cc

```diff
@@ -419,6 +419,39 @@ TEST(NhwcTransformerTests, ConvGlobalAveragePool) {
   NhwcTransformerTester(build_test_case, check_nhwc_graph);
 }
 
+TEST(NhwcTransformerTests, ConvSplit) {
+  auto build_test_case = [&](NhwcTestHelper& helper) {
+    auto* input_arg = helper.MakeInput<uint8_t>({1, 23, 13, 13});
+    auto* conv_output_arg = helper.MakeIntermediate();
+    auto* split_output1_arg = helper.MakeIntermediate();
+    auto* split_output2_arg = helper.MakeIntermediate();
+    auto* qladd_output_arg = helper.MakeIntermediate();
+    auto* output_arg = helper.MakeOutput();
+
+    Node& conv_node = helper.AddQLinearConvNode<uint8_t>(input_arg, .01f, 135,
+                                                         {64, 23, 3, 3}, .02f, 126,
+                                                         conv_output_arg, .37f, 131);
+    conv_node.AddAttribute("pads", std::vector<int64_t>{1, 1, 1, 1});
+    Node& split_node = helper.AddNode("Split", {conv_output_arg}, {split_output1_arg, split_output2_arg});
+    split_node.AddAttribute("axis", static_cast<int64_t>(1));
```

Comment:
For the split test case, because there are several test conditions in the optimizer, would it be better if the test did something like "for (int64_t axis = -4; axis < 4; axis++) {" and test each axis value?

Reply:
Fixed.